internal/controllers/control/cassandratask_controller.go

```diff
@@ -815,6 +814,13 @@ func (r *CassandraTaskReconciler) reconcileEveryPodTask(ctx context.Context, dc
 					<-jobRunner
 				}()
 
+				if taskConfig.SyncFeature != "" {
```

Comment:
issue: This feature is not tested, not for actually working or for regression. It would require

a) Test that checks that existing jobs without SyncFeature set still work as is
b) Test that checks that new job with SyncFeature (but no server support) is correctly failing
c) Test that checks that new job with SyncFeature and server support is correctly called

Easiest way would be to modify your existing test to verify against Feature endpoint and add SyncFeature in its definition and run it. The a) is taken care of running a sync against featureset endpoint and still working and b) requires a new test.

Reply:
I think all of these are now present.